src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -131,12 +135,73 @@ private function getOrderToolbarButtons(): array
         return $toolbarButtons;
     }
 
-    //@todo: wip
+    /**
+     * Places an order from BO
+     *
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller'))")
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function placeAction(Request $request)
+    {
+        $summaryForm = $this->createForm(CartSummaryType::class);
+        $summaryForm->handleRequest($request);
+
+        if ($summaryForm->isSubmitted() && $summaryForm->isValid()) {
+            $formData = $summaryForm->getData();
+            try {
+                $orderId = $this->getCommandBus()->handle(new AddOrderFromBackOfficeCommand(
+                    (int) $formData['cart_id'],
+                    $this->getContext()->employee->id,
+                    $formData['order_message'],
```

Comment:
Be careful of non required data, if sent empty the request don't contain them thus the form data might not have this field which would result with an access error Either check the field presence or set default data in the FormType